.github/workflows/integration.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_REGION=${{ secrets.AWS_REGION }}
           JWT_SECRET=${{ secrets.JWT_SECRET }}
-          DATABASE_URL=${{ secrets.DATABASE_URL }}
+          DATABASE_URL=${{ github.ref == 'refs/heads/main' && secrets.PROD_MAPA_DATABASE_URL || secrets.STG_MAPA_DATABASE_URL }}
```

Comment:
por enquanto a gente precisa manter as do BONDE! mas podemos deixar esse PR em aberto pra quando a gente for fazer a migration 🤔